src/table/components/TableHeadWrapper.jsx

```diff
@@ -42,6 +42,11 @@ function TableHeadWrapper({
               setFocusedCellCoord,
             });
           };
+          const handleMouseDown = (evt) => {
```

Comment:
their are multiple mouse downs in this file, this name is too generic.

Reply:
Fixed